.github/workflows/build.yml

```diff
@@ -15,23 +15,23 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@f43a0e5ff2bd294095638e18286ca9a3d1956744 # v3
```

Comment:
Miksi actionseissä on käytetty SHA-arvoa eikä versionumeroa? 
Dokumentaatiossa (https://docs.github.com/en/actions/how-tos/write-workflows/choose-what-workflows-do/find-and-customize-actions#using-shas) lukee "this approach means you will not automatically receive updates for an action, including important bug fixes and security updates."